src/functions-reference/multivariate_discrete_distributions.Rmd

```diff
@@ -68,48 +68,48 @@ scale.
 
 ### Probability mass function
 
-If $K \in \mathbb{N}$, $N \in \mathbb{N}$, and $\text{softmax}^{-1}(\theta) \in
+If $K \in \mathbb{N}$, $N \in \mathbb{N}$, and $\text{softmax}(\theta) \in
 \text{$K$-simplex}$, then for $y \in \mathbb{N}^K$ such that
 $\sum_{k=1}^K y_k = N$, \[
-\text{MultinomialLogit}(y|\theta) = \text{Multinomial}(y|\text{softmax}^{-1}(\theta)) =
-\binom{N}{y_1,\ldots,y_K} \prod_{k=1}^K [\text{softmax}^{-1}(\theta)_k]^{y_k}, \] where the
+\text{MultinomialLogit}(y \mid \gamma = \text{log\_softmax}(\theta)) = \text{Multinomial}(y \mid \text{softmax}(\theta)) =
+\binom{N}{y_1,\ldots,y_K} \prod_{k=1}^K [\text{softmax}(\theta)_k]^{y_k}, \] where the
```

Comment:
Line 74 should just be:

```
MultinomialLogit(y | gamma)
= Multinomial(y | softmax(gamma))
= ... formula ...
```

The only problem with the original was a stray `^{-1}`.